charts/nri-prometheus/values.yaml

```diff
@@ -105,10 +105,21 @@ config:
   require_scrape_enabled_label_for_nodes: true
 
   # Number of worker threads used for scraping targets.
-  # For large clusters with many endpoints, increase to 10.
+  # For large clusters with many (>100) endpoints, slowly increase until scrape
```

Comment:
In the documentation we specify 400 as a safe value, we should be consistent

Reply:
https://docs.newrelic.com/docs/integrations/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations-large-kubernetes-environments